src/main/java/com/aws/greengrass/clientdevices/auth/metrics/MetricsEmitter.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.metrics;
+
+import com.aws.greengrass.logging.api.Logger;
+import com.aws.greengrass.logging.impl.LogManager;
+import com.aws.greengrass.telemetry.PeriodicMetricsEmitter;
+import com.aws.greengrass.telemetry.impl.Metric;
+import com.aws.greengrass.telemetry.impl.MetricFactory;
+
+import java.util.List;
+import javax.inject.Inject;
+
+public class MetricsEmitter extends PeriodicMetricsEmitter {
+    public static final Logger logger = LogManager.getLogger(MetricsEmitter.class);
+    public static final String NAMESPACE = "ClientDeviceAuth";
```

Comment:
```suggestion
    private static final String NAMESPACE = "ClientDeviceAuth";
```
keep scope low until we need it higher